features/step_definitions/register.rb

```diff
@@ -0,0 +1,11 @@
+Given(/^that I can see the menu page$/) do
+  visit '/'
+end
+
+When(/^I click the "([^"]*)" link$/) do |link_text|
+  click_link(link_text)
+end
+
+Then(/^I want to be "([^"]*)" to the register customer page$/) do |arg1|
```

Comment:
Lets give `arg1` a more meaningful name and btw why do we have it while we are not using it in the code body?